### PR TITLE
Portfolio Mobile version  completed

### PR DESCRIPTION
**Pull request by mohashyne**
- [ ]  Add Work/Projects section
- [ ]  Add skills section (about myself & skills)
- [ ]  Add Contact section
- [ ]  Add styling to all sections

### CODE REVIEW

File: dist/css/style.css [hunk 1]
```diff
@@ -123,3 +123,507 @@ h1 {
...
+}
+
+.client {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.8rem 0;
+  width: 5.5rem;
+  height: 3.2rem;
+  font-family: 'Poppins', sans-serif;
```
Comment (Qoosim) [inline dist/css/style.css:207]: - [ ] Great job so far. Before you can use the font-family used in the design which is  `Poppins`, you need to add it to he head section of your `index.html` or import it inside your `style.css` file. Kindly import the font from [googlefont](https://fonts.google.com/specimen/Poppins?query=poppins). currently it does not exist in your file, so the one used is the fallback option which is sans-serif. Kindly apply it to every other areas where applicable.

File: dist/index.html [hunk 2]
```diff
@@ -57,6 +57,222 @@ <h2 href="#" class="connect">LET'S CONNECT</h2>
...
+                  </div> 
+            </section>
+
+            <section  class="my-skills">
+                <div class="about "> 
+                    <div class="about-self">
+                        <div><h2 class="heading">About  Myself</h2></div>
+                        <div><p class="info">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p></div>
+                        <h3 href="#" class="connect">LET'S CONNECT</h3>
+                   <ul class="social-media">
```
Comment (Qoosim) [inline dist/index.html:168]: - [ ] Kindly adjust the style for the `LET'S CONNECT` to be on a single line as we have it on the design.

### Your Design
your-let-connect


### Figma Design
temp-let-connect

Comment (Qoosim) [inline dist/index.html:168]: - [ ] Kindly use the font-family used in the design for every text on this section as I've mentioned above about the `Poppins` font that you need to import from google fonts.

### Your Design
your-let-connect


### Figma Design
temp-let-connect

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @mohashyne, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**